packages/core/src/tracing/spanBuffer.js

```diff
@@ -87,6 +89,10 @@ const batchBucketWidth = 18;
 
 /** @type {BatchingBucketMap} */
 const batchingBuckets = new Map();
+/**
+ * @type {Object}
```

Comment:
issue: Here we are using type Object and in packages/core/src/tracing/index.js we define the actual type.